apps/juxtaposition-ui/src/webfiles/ctr/js/debug.js

```diff
@@ -1,17 +1,3 @@
-if (typeof cave !== 'undefined') {
-	window.addEventListener('error', handleError, true);
-
-	function handleError(evt) {
-		if (evt.message) { // Chrome sometimes provides this
-			alert('error: ' + evt.message + ' at linenumber: ' + evt.lineno + ' of file: ' + evt.filename);
-		} else {
-			if ((evt.srcElement || evt.target).toString().indexOf('HTMLImageElement') !== -1) {
-				return;
-			}
-			alert('error: ' + evt.type + ' from element: ' + (evt.srcElement || evt.target));
-		}
-	}
-}
```

Comment:
What did this do?

Reply:
On 3DS consoles (cave not undefined) it would show an alert dialog for JavaScript errors. I had never seen nor heard of anyone seeing these alerts, and the plan was to stop shipping this file to 3DSes anyway, so I yoinked it.
We could add it back into juxt.js if it serves a useful debugging format.